doc/examples/transform/plot_register_align.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+from scipy import ndimage as ndi
+from matplotlib import pyplot as plt
+
+from skimage.data import camera
+from skimage.transform import register_affine
+from skimage import measure
+
+callback_arr = []
+callback = lambda i, m: callback_arr.append((i,m))
+r = 0.12
+out = np.empty((3, 3))
```

Comment:
I'm not sure 'out' is a very helpful variable name here, particularly since it's not actually the output from your function.

It might be more helpful to have:
out -> matrix_transform
cam -> reference_image
start -> target_image
trans -> output_image / registered_image / something like that.